server/route.go

```diff
@@ -21,70 +22,128 @@ var (
 	errTimeout    = errors.New("Channel sending timeout")
 )
 
+type RouteOptions struct {
+	Path protocol.Path
+
+	// channel buffer size
+	Size int
```

Comment:
since it's a public field, do you think renaming it to what is written in the comment is helpful, and then the comment becomes unnecessary? :)
=> ChannelBufferSize